dev.html

```diff
@@ -2799,6 +2952,296 @@ <h2>Conditions</h2>
+
+      Quoted triples can be created by referencing the mapping identifier using the <code>quoted</code> keyword.
+      It is also possible to create non-asserted quoted triples using the <code>quotedNonAsserted</code> keyword.
+
+      <section>
+        <h2>Keys</h2>
+
+        <dl>
+          <dt><dfn>quoted</dfn></dt>
+          <dd>Quoted mapping.<br />
+            Required profiles: RMLStar. <br />
+            Datatype: Mapping reference.
+          </dd>
+          <dt><dfn>quotedNonAsserted</dfn></dt>
+          <dd>Non-asserted quoted mapping.<br />
+            Required profiles: RMLStar. <br />
+            Datatype: Mapping reference.
+          </dd>
+        </dl>
+      </section>
+
+    
+      <section>
+        <h2>Examples</h2>
+
+
+      <div id="tabs-60" class="nanotabs">
+        <ul>
+          <li><a href="#tab60-1" class="selected">YARRRML</a></li>
+          <li><a href="#tab60-2">RML</a></li>
+        </ul>
+        <div id="tab60-1">
+          <pre id="example-fn-oneline-no-prefix" class="example" title="Quoted mapping in object">
+            mappings:
+              person:
+                subject: http://example.org/person/$(Name)
+                predicateobjects:
+                 - predicates: ex:confirms
+                   objects:
+                    - quoted: student
+              student:
+                subject: http://example.org/student/$(Student)
+                predicateobjects:
+                 - [a, ex:Student]
+
+          </pre>
+        </div>
+        <div id="tab60-2">
+            <pre id="example-fn-oneline-rml-no-prefix" class="example" title="Quoted mapping in object">
+              <#person>
+                a rr:TriplesMap ;
+                rml:subjectMap [
+                  rr:template "http://example.com/person/{Name}"
+                ] ;
+                rr:predicateObjectMap [
+                  rr:predicate ex:confirmse ;
+                  rml:objectMap [
+                    rml:quotedTriplesMap <#student>;
+                  ];
+                ] .
+
+              <#student>
+                a rr:TriplesMap ;
+                rml:subjectMap [
+                  rr:template "http://example.org/student/{Student}";
+                ] ;
+                rr:predicateObjectMap [
+                  rr:predicate rdf:type ;
+                  rml:object ex:Student
+                ] .
+            </pre>
+        </div>
+      </div>
+
+
+      <code>Quoted</code> mappings can be also defined in subjects
+
+      <div id="tabs-61" class="nanotabs">
+        <ul>
+          <li><a href="#tab61-1" class="selected">YARRRML</a></li>
+          <li><a href="#tab61-2">RML</a></li>
+        </ul>
+        <div id="tab61-1">
+          <pre id="example-fn-oneline-no-prefix" class="example" title="Quoted mapping in subject">
+            mappings:
+              person:
+                subject:
+                 - quoted: student
```

Comment:
Same comment here: either you introduce a new key, or you use the `~[specifier]` construct. I personally prefer - given you refer to a mapping in `mapping`, `quoted`, and `unquoted` - the specifier

Reply:
My question is, can both co-exist? and we treat `~[specifier]` as kind of shortcut?